### PR TITLE
Misc AmCharts Changes (2) (1.10.0)

### PR DESCRIPTION
**Pull request by mlzummo**
Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

File: src/amchart/Column.js [hunk 1]
```diff
@@ -44,6 +44,14 @@
         CommonSerial.prototype.updateChartOptions.apply(this, arguments);
         var context = this;
 
+        if (this._rangeType === "candle-ohlc") {
```
Comment (mlzummo) [inline src/amchart/Column.js]: i dont remember this? .... looks like 1.10 isnt up to date with some stuff?

File: src/amchart/CommonSerial.js [hunk 2]
```diff
@@ -383,6 +392,35 @@
 
         this.amFormatColumns();
 
+                if (!this._chart.colors) {
```
Comment (mlzummo) [inline src/amchart/CommonSerial.js]: remove

**Comment by mlzummo** [comment]
ready to pull

**Comment by GordonSmith** [comment]
@mlzummo Please Rebase

**Comment by mlzummo** [comment]
rebased

**Comment by GordonSmith** [comment]
Closing as it has no real changes.

**Pull request closed without merging**